### DISCUSSION TITLE
Linux 上有什么好用的v2的GUI程序吗

### DISCUSSION THREAD

**Discussion by Mixlining** [v2fly/v2ray-core#1470] [Q&A]
ubuntu新手，用命令行总是卡手，请问有什么好用的GUI吗

**Comment by kallydev** [comment]
> ubuntu新手，用命令行总是卡手，请问有什么好用的GUI吗

[Qv2ray](https://github.com/Qv2ray/Qv2ray).

https://www.v2fly.org/awesome/tools.html
